internal/pwru/output.go

```diff
@@ -120,7 +181,50 @@ func (o *output) Print(event *Event) {
 	case "arm64":
 		addr = event.Addr
 	}
+	return addr
+}
+
+func getTupleData(event *Event) (tupleData string) {
+	tupleData = fmt.Sprintf("%s:%d->%s:%d(%s)",
```

Comment:
Could we get this as JSON too?

Reply:
I'm not sure what you mean, but running the following command:
`./pwru --output-tuple --output-json`

outputs:
```
{"cpu":0,"func":"tcp_urg","process":"<empty>:(2838)","skb":"0xffff10988cec8f00","tuple":"198.19.248.1:32611->198.19.248.2:2223(tcp)"}
```

jq works on top of it:
```
./pwru --output-tuple --output-json | jq .tuple
"127.0.0.1:51626->127.0.0.1:2379(tcp)"
"127.0.0.1:51626->127.0.0.1:2379(tcp)"
"127.0.0.1:51626->127.0.0.1:2379(tcp)"
"127.0.0.1:2379->127.0.0.1:51626(tcp)"
```

Reply:
I meant:
```
"tuple": { "sip": ..., "dip": ..., "proto": ..., "sport": ..., "dport": ...}
```

Reply:
Thank you for explaining. Now I get what you meant. I will implement it.

Reply:
Implemented, but I think we should refactor and move the pieces of the output to structs. What do you think?


```
{"cpu":2,"func":"__copy_skb_header","process":"containerd-shim:1832","skb":"0xffff1098826d40e0","tuple":{"dip":"1.1.1.1","dport":443,"proto":"tcp","sip":"198.19.249.52","sport":37230}}
```

Reply:
SGTM!

Reply:
So do you think we should move it or merge like it is right now?

Reply:
Refactor and move.

Reply:
done